src/lib.rs

```diff
@@ -133,16 +133,12 @@ impl Isolate {
     pub fn post(&self, msg: impl IntoDart) -> bool {
         if let Some(func) = POST_COBJECT.load(Ordering::Relaxed) {
             unsafe {
-                let boxed_msg = Box::new(msg.into_dart());
```

Comment:
I believe that the boxing here is required to get a fat pointer.

Reply:
Reference can be coerced to pointers and this is perfectly fine:

```rust
    let mut foo = String::new();
    let foo_ptr: *const String = &foo;
    let foo_mut_ptr: *mut String = &mut foo;
```

It is basically a pointer to something on stack. I don't see any reason to do boxing here since you are not transferring ownership to Dart and there are no lifetime requirements (box is dropped before exiting the function). This PR does not change how things work, thats just minor performance improvement.